src/directors/directors.json

```diff
@@ -0,0 +1,540 @@
+{
+  "$schema": "./schema.json",
+  "directors": [
+    {
+      "first_name": "Юрий",
```

Comment:
Could you please add id for every person?
And you can use --amend option for git commit to avoid adding new commits

Reply:
Okay I will, but I would suggest you to avoid forcing commits in a repository of a project with multiple collaborators.

Reply:
for you branch it's okay